sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -184,6 +184,7 @@ pi_result piDeviceGetInfo(pi_device device, pi_device_info paramName,
   case PI_DEVICE_INFO_GPU_SLICES:
   case PI_DEVICE_INFO_GPU_SUBSLICES_PER_SLICE:
   case PI_DEVICE_INFO_GPU_EU_COUNT_PER_SUBSLICE:
+  case PI_DEVICE_INFO_GPU_HW_THREADS_PER_EU:
```

Comment:
There is actually an extension, that adds these queries for OpenCL: https://github.com/KhronosGroup/OpenCL-Docs/blob/master/extensions/cl_intel_device_attribute_query.asciidoc
I plan to add these queries to pi_opencl all together a bit later.

Reply:
'A bit later' is prolonged since even if the extension is public - it's yet not in OpenCL register and the headers and hence my local build for this implementation is failing.

Reply:
Maybe add a TODO comment naming the plans

Reply:
Done